react/codecamp-tutorial/README.md

```diff
@@ -1,3 +1,22 @@
-# React Tutorial
+# React Meme Generator
```

Comment:
Oops. This is the wrong README. It should have been one in meme generator directory.